config/default.yml

```diff
@@ -685,6 +685,15 @@ Naming/PredicateName:
   Exclude:
     - 'spec/**/*'
 
+Naming/UncommunicativeBlockParamName:
+  # Parameter names may be equal to or greater than this value
+  MinParamNameLength: 1
```

Comment:
Btw, with such a default will this ever flag anything? :-)

Reply:
It will only flag BlockParams that end in numbers or contain capital letters. It will never flag length. But, should users want to enforce a 2 or 3 char min length, they have the ability to.